Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -507,8 +510,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Checks if the user executes LDAP queries on a regular basis from
-        one or from multiple hosts, daily.
+      description: Checks if the user executes LDAP queries on a regular basis from one or from multiple hosts, daily.
```

Comment:
Checks whether the user executes LDAP daily queries on a regular basis from one or more hosts.